content/blog/2017/2017-04-10-Jenkins-has-upgraded-to-Java-8.adoc

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: "Jenkins Now Requires Java 8 To Run Starting With 2.54"
+tags:
+- java8
+- upgrade
+author: batmat
+---
+
+:imagesdir: /images/post-images/2017-04-java8
+
+image:lambda.png[role="right"] As link:/blog/2017/01/17/Jenkins-is-upgrading-to-Java-8/[announced some weeks ago], starting with version link:https://jenkins.io/changelog/#v2.54[Jenkins **2.54**], the Java runtime requirement has been updated to Java 8.
```

Comment:
Use this one: https://upload.wikimedia.org/wikipedia/commons/thumb/8/8f/Orange_lambda.svg/1200px-Orange_lambda.svg.png :)